src/DurableTask.Netherite/StorageLayer/Faster/AzureBlobs/BlobManager.cs

```diff
@@ -1459,5 +1458,10 @@ internal async Task FinalizeCheckpointCompletedAsync()
                     });
             }
         }
+
+        public void CheckpointVersionShift(long oldVersion, long newVersion)
+        {
+            throw new NotImplementedException();
+        }
```

Comment:
Any pointers on how to implement this method? Where should we update the version?

Reply:
Also worth checking this w/ FASTER team, however, from looking at the source: https://github.com/microsoft/FASTER/blob/0116754bcf0857e614e2b7ea042b8db563cb26cc/cs/src/core/Index/Recovery/ICheckpointManager.cs#L61-L66

it appears this is just a callback for us, that we may ignore. I.e, we could use this callback to verbose log that the checkpoint version has increased, but it doesn't have to do real work. Again, just this is just a best guess though :)

Reply:
I believe it is safe to treat this callback as a no-op (which is what David is suggesting) since the default commit manager in FASTER (DeviceLogCommitCheckpointManager) also treats this as a no-op, so it can't really be important.

Reply:
That's true as far as I can see; it's called during the state machine processing but nothing depends on the result. So just treat as a no-op